modules/apps/asset/asset-tags-admin-web/src/main/resources/content/Language_fa.properties

```diff
@@ -2,7 +2,7 @@ a-tag-with-that-name-already-exists=یک تگ با این نام هم اکنون
 add-tag=اضافه کردن صفحه
 are-you-sure-you-want-to-merge-x-into-x=با "{1}" ادغام شوند؟ در این‌صورت تمام آیتم‌هایی که برچسب "{0}" خورده‌اند برچسب "{1}" می‌گیرند. "{0}" آیا مطمئن هستید که می‌خواهید
 into-this-tag=به این تگ:
-javax.portlet.description.com_liferay_asset_tags_admin_web_portlet_AssetTagsAdminPortlet=تگ‌ها گروه‌بندی مطالب را از طریق گروه‌های لغات و اصطلاحات انعطاف‌پذیر، فعال می‌کند. مدیر می‌تواند تمام تگ‌ها را مدیریت کند، مجوز دسترسی صادر کند و گزین‌های پیشرفته را تنظیم کند. کاربران می‌توانند تمام تگ‌هایشان را مدیریت کنند، تنظیمات تگ را تعریف کرده و دسته‌بندی کنند تا بتوانند از طبق‌بندی محتوا استفاده کنند.
+javax.portlet.description.com_liferay_asset_tags_admin_web_portlet_AssetTagsAdminPortlet=تگ‌ها گروه‌بندی مطالب را از طریق گروه‌های لغات و اصطلاحات انعطاف‌پذیر، فعال می‌کند. مدیر می‌تواند تمام تگ‌ها را مدیریت کند، مجوز دسترسی صادر کند و گزین‌های پیشرفته را تنظیم کند. کاربران می‌توانند تمام تگ‌هایشان را مدیریت کنند، تنظیمات تگ را تعریف کرده و دسته‌بندی کنند تا بتوانند از طبق‌بندی محتوا استفاده کنند.
 javax.portlet.long-title.com_liferay_asset_tags_admin_web_portlet_AssetTagsAdminPortlet=برچسب‌ها
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines. I suspect it should be `0x20` (space).